CMakeLists.txt

```diff
@@ -1,10 +1,19 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(kuka_lwr4_description)
-find_package(catkin REQUIRED)
+
+find_package(catkin REQUIRED COMPONENTS xacro)
 
 catkin_package(CATKIN_DEPENDS xacro)
 
-foreach(dir calibration defs meshes robots)
+# generate and install URDFs from xacro files
+# source: http://wiki.ros.org/xacro#Building_from_CMakeLists.txt
+
+# find all LWR robots with tools
+file(GLOB xacro_files ${CMAKE_CURRENT_SOURCE_DIR}/robots/lwr_*.urdf.xacro)
```

Comment:
List individual files here. GLOB is convenient but it will hide which files get compiled. It is easy to switch branch and forget to include new files.